index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+  <body>
+      <header>
+        <div class="toolbar">
+            <div>
+                <h1>Ishmael</h1>
+            </div>
```

Comment:
- [ ] This is the 'Logo' part of the assignment which looks good however it should be in an anchor tag `<a>`  and it can't be an `<h1>` tag as this one is reserved for the main title e.g. `<h1>Hey There. I’m Alex</h1>`